data/adapters/BlueSTARR_variant_elements_adapter.py

```diff
@@ -0,0 +1,54 @@
+import csv
+import json
+from adapters.helpers import build_variant_id_from_hgvs, build_regulatory_region_id
+from typing import Optional
+
+from adapters.writer import Writer
+# Example line from file from IGVFFI1236SEPK
+# chr10	100005234	100005491	0.126	NC_000010.11:100005302:A:C
+# chr10	100005234	100005491	0.129	NC_000010.11:100005339:A:C
+# chr10	100005234	100005491	-0.136	NC_000010.11:100005425:A:T
+
+
+class BlueSTARRVariantElement:
+    ALLOWED_LABELS = ['variant_genomic_elements']
+    SOURCE = 'IGVF'
+    SOURCE_URL = 'https://data.igvf.org/prediction-sets/IGVFDS0257SDNV/'
+
+    def __init__(self, filepath, label='variant_element', writer: Optional[Writer] = None, **kwargs):
+        if label not in BlueSTARRVariantElement.ALLOWED_LABELS:
+            raise ValueError('Invalid label. Allowed values: ' +
+                             ','.join(BlueSTARRVariantElement.ALLOWED_LABELS))
+
+        self.filepath = filepath
+        self.writer = writer
+
+    def process_file(self):
+        self.writer.open()
+
+        with open(self.filepath, 'r') as bluestarr_tsv:
+            bluestarr_tsv = csv.reader(bluestarr_tsv, delimiter='\t')
+            next(bluestarr_tsv)
+            for row in bluestarr_tsv:
+                _id = build_variant_id_from_hgvs(row[4])
+                element_id = build_regulatory_region_id(row[0], row[1], row[2])
```

Comment:
wait for reloading ccre (key should be build_regulatory_region_id(row[0], row[1], row[2]) + '_ IGVFFI7195KIHI'

Reply:
ccre have been reloaded with element coordinates. I think the element ID should still be just the ID without the file accession, no?

Reply:
We did append file accession to the element id, in case the same region is occurred in multiple datasets (rare case though), you can check the adapter code here: https://github.com/IGVF-DACC/igvf-catalog/blob/20c7b0e35d6378fa3ddf48ecd9c5751e8db4e34b/data/adapters/ccre_adapter.py#L58, or look at some examples in arangodb.

Reply:
key should be:
build_regulatory_region_id(row[0], row[1], row[2], 'candidate_cis_regulatory_element') + '_IGVFFI7195KIHI'

Reply:
fixed